modules/mixed-arch-creating-arm64-bootimage.adoc

```diff
@@ -0,0 +1,121 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/cluster-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="mixed-arch-creating-arm64-bootimage"]
+
+= Creating an `arm64` bootimage using the Azure image gallery
+ 
+Before you modify the machine set, you must manually generate the `arm64` bootimage. 
+ 
+.Prerequisites
+
+* Installed the Azure CLI (`az`).
+* Configured your Azure portal account.
+* Created a single-architecture Azure installer-provisioned cluster with the heterogeneous installer binary. 
+
+.Procedure
+. Login to your Azure account: 
++
+[source,terminal]
+----
+az login
+----
+. Create a storage account and upload the `arm64` virtual hard disk (VHD) to your storage account:
++
+[source,terminal]
+----
+az storage account create -n <storage-account> -g <resource-group> -l westus --sku Standard_LRS 
+----
++
+Create a storage container using the storage account you generated:
++
+[source,terminal]
++
+----
+az storage container create -n <container> --account-name <storage-account>
+----
+. You must use the OpenShift installer JSON file to extract the `URL` and `arch64` VHD name:
+.. Extract the `URL` field set it to `RHCOS_vhd_origin_url`:
++
+[source,terminal]
+----
+RHCOS_vhd_origin_url=$(./openshift-install coreos print-stream-json | jq -r '.architectures.aarch64."rhel-coreos-extensions"."azure-disk".url')
+----
+.. Extract the `aarch64` VHD name and set it to `Blob_name`:
++
+[source,terminal]
+----
+Blob_name=rhcos-$(./openshift-install coreos print-stream-json | jq -r '.architectures.aarch64."rhel-coreos-extensions"."azure-disk".release')-azure.aarch64.vhd
+----
+. Generate a shared access signature (SAS) token. Use this token to upload the {op-system} VHD to your storage container: 
++
+[source,terminal]
+----
+end=`date -u -d "30 minutes" '+%Y-%m-%dT%H:%MZ'`
+sas=`az storage container generate-sas -n <container> --account-name <storage-account> --https-only --permissions dlrw --expiry $end -o tsv`
+----
+. Copy the {op-system} VHD into the storage container:
++
+[source, terminal]
+----
+az storage blob copy start --account-name $<storage-account> --sas-token "$sas" \
+ --source-uri "$RHCOS_vhd_origin_URL" \
+ --destination-blob "$Blob_name" --destination-container $<container>
+----
++
+You can check the status of the of copying process with the following command:
++
+[source,terminal]
+----
+"az storage blob show -c $<container> -n $Blob_name --account-name $<storage-account> | jq .properties.copy"
+----
++
+.Example output
+[source,terminal]
+----
+ "completionTime": null,
+ "destinationSnapshot": null,
```

Comment:
This snippet is missing the brackets at the beginning and the end. It is a json object. Not sure it's intended